### PR TITLE
added social icons for header

### PR DESCRIPTION
**Pull request by Muljayan**
### Before
image

### After

image

### CODE REVIEW

File: src/components/FixMeNavbar/FixMeNavbar.tsx [hunk 1]
```diff
@@ -74,38 +64,16 @@ export default class FixMeNavbar extends React.Component<
...
-                    />
-                  </svg>
+                <a className="nav-link" href="https://github.com/ossn">
```
Comment (alexdor) [inline src/components/FixMeNavbar/FixMeNavbar.tsx]: Could you use the component OutboundLink here?

File: src/components/FixMeNavbar/FixMeNavbar.tsx [hunk 2]
```diff
@@ -48,24 +49,13 @@ export default class FixMeNavbar extends React.Component<
                   About
                 </NavLink>
               </NavItem>
-              {/* facebook
-               <NavItem>
+
+              <NavItem>
                 <a className="nav-link" target="_blank" href="#">
```
Comment (alexdor) [inline src/components/FixMeNavbar/FixMeNavbar.tsx]: Could you switch this for the OutboundLink component?

**Comment by Muljayan** [comment]
@alexdor  the requested changes are done . Also i've used the react-icons package because we can easily embed customizable SVG icons anywhere on the site. Also it avoids clutter in the code which usually occurs when embedding long SVG codes.

File: src/components/FixMeNavbar/FixMeNavbar.tsx [hunk 3]
```diff
@@ -23,7 +24,7 @@ export default class FixMeNavbar extends React.Component<
   };
   public render() {
     const { white } = this.props;
-    const fill = white ? "#282a3a" : "#FAFAFA";
+    // const fill = white ? "#282a3a" : "#FAFAFA";
```
Comment (alexdor) [inline src/components/FixMeNavbar/FixMeNavbar.tsx]: could you delete this line completely?

File: src/components/FixMeNavbar/FixMeNavbar.tsx [hunk 4]
```diff
@@ -48,24 +49,19 @@ export default class FixMeNavbar extends React.Component<
...
-                      fill={fill}
-                      fillRule="evenodd"
-                      d="M15.99 0H2.01C1.452 0 .977.198.587.595A1.95 1.95 0 0 0 0 2.011v13.998c0 .546.195 1.015.586 1.405.39.39.865.586 1.425.586H9v-6.99H7.009V8.532H9v-2.05c0-1.08.31-1.965.927-2.655.619-.69 1.565-1.034 2.84-1.034h1.797v2.616h-1.19c-.495 0-.847.153-1.055.459a1.69 1.69 0 0 0-.312.966v1.698h2.557l-.566 2.48h-1.991V18h3.982a1.94 1.94 0 0 0 1.425-.586c.39-.39.586-.859.586-1.405V2.01a1.95 1.95 0 0 0-.586-1.416A1.924 1.924 0 0 0 15.99 0z"
-                    />
-                  </svg>
-                </a>
+
+              <NavItem>
+                <OutboundLink
+                  aria-label="FACEBOOK"
```
Comment (alexdor) [inline src/components/FixMeNavbar/FixMeNavbar.tsx]: Could you change the label to "fixme's facebook"?

File: src/components/FixMeNavbar/FixMeNavbar.tsx [hunk 5]
```diff
@@ -48,24 +49,19 @@ export default class FixMeNavbar extends React.Component<
...
+                  eventLabel="Facebook on menu clicked"
+                >
+                  <FaFacebook />
+                </OutboundLink>
               </NavItem>
-             */}
+
               <NavItem>
                 <OutboundLink
                   aria-label="TWITTER"
```
Comment (alexdor) [inline src/components/FixMeNavbar/FixMeNavbar.tsx]: Could you change the label to "fixme's twitter"?

File: src/components/FixMeNavbar/FixMeNavbar.tsx [hunk 6]
```diff
@@ -74,38 +70,21 @@ export default class FixMeNavbar extends React.Component<
...
-                    />
-                  </svg>
-                </a>
+                <OutboundLink
+                  aria-label="GITHUB"
```
Comment (alexdor) [inline src/components/FixMeNavbar/FixMeNavbar.tsx]: Could you change the label to "fixme's github"?

File: package.json [hunk 7]
```diff
@@ -35,6 +35,7 @@
     "react-dom": "^16.5.2",
     "react-ga": "^2.5.3",
     "react-helmet": "^5.2.0",
+    "react-icons": "^3.2.1",
```
Comment (alexdor) [inline package.json]: Could you also include the types for react-icons? https://www.npmjs.com/package/@types/react-icons

**Comment by Muljayan** [comment]
@alexdor the requested changes have been made.

**Comment by alexdor** [comment]
Looks good @Muljayan  : )

**Pull request merged**